pkg/customers/customers.go

```diff
@@ -486,6 +486,7 @@ func (r *sqlCustomerRepository) updateCustomer(c *client.Customer, organization
 	if err != nil {
 		return err
 	}
+	defer tx.Rollback()
```

Comment:
`tx.Rollback()` was being called multiple times but it only needs to be called once in a `defer` since a rollback won't occur if the transaction is successfully commited.

Source: https://stackoverflow.com/questions/46421602/why-defer-a-rollback

Reply:
TIL. I didn't know Rollback checked if the transaction was successful or not.